pkg/controller/operands/kubevirt_test.go

```diff
@@ -941,6 +941,34 @@ Version: 1.2.3`)
 					})
 				})
 
+				It("should add the SRIOVLiveMigration feature gate if it's set in HyperConverged CR", func() {
+					// one enabled, one disabled and one missing
+					hco.Spec.FeatureGates = hcov1beta1.HyperConvergedFeatureGates{
+						SRIOVLiveMigration: &enabled,
+					}
+
+					existingResource, err := NewKubeVirt(hco)
+					Expect(err).ToNot(HaveOccurred())
+					By("KV CR should contain the HotplugVolumes feature gate", func() {
+						Expect(existingResource.Spec.Configuration.DeveloperConfiguration).NotTo(BeNil())
+						Expect(existingResource.Spec.Configuration.DeveloperConfiguration.FeatureGates).To(ContainElement(kvSRIOVLiveMigration))
+					})
+				})
+
+				It("should not add the SRIOVLiveMigration feature gate if it's disabled in HyperConverged CR", func() {
+					// one enabled, one disabled and one missing
+					hco.Spec.FeatureGates = hcov1beta1.HyperConvergedFeatureGates{
+						SRIOVLiveMigration: &disabled,
+					}
+
+					existingResource, err := NewKubeVirt(hco)
+					Expect(err).ToNot(HaveOccurred())
+					By("KV CR should contain the HotplugVolumes feature gate", func() {
+						Expect(existingResource.Spec.Configuration.DeveloperConfiguration).NotTo(BeNil())
+						Expect(existingResource.Spec.Configuration.DeveloperConfiguration.FeatureGates).ToNot(ContainElement("SRIOVLiveMigration"))
+					})
+				})
+
```

Comment:
Please add this flag to the tests in `Context("test feature gates in KV handler"`

Reply:
Done